Sources/StreamChatUI/ChatChannelList/ChatChannelListVC.swift

```diff
@@ -105,6 +135,18 @@ open class _ChatChannelListVC<ExtraData: ExtraDataTypes>: _ViewController,
                 height: 64
             )
         }
+    
+        (errorOccurredNotificationView as? ChatChannelListErrorView)?.buttonAction = { [weak self] in
```

Comment:
Why don't you make `ChatChannelListErrorView` a `ViewWithAction` and use the protocol conformance?

Reply:
Discussed internally, it actually doesn't make sense to wrap this in abstraction because if our customers will implement their own ErrorView.